KsApi/models/graphql/adapters/Project+FetchProjectRewardsByIdQueryDataTests.swift

```diff
@@ -84,4 +85,21 @@ final class Project_FetchProjectRewardsByIdQueryDataTests: XCTestCase {
     XCTAssertEqual(localPickup.displayableName, "San Jose, CA")
     XCTAssertNil(rewards.first?.localPickup)
   }
+
```

Comment:
I decided to just add in an extra reward and test the shippingRulesExpanded instead of needing to recreate the entire test. This tests that the new shipping rules are being handled as expected without me needing to spend a ridiculous amount of time on this test.